### ISSUE TITLE
FATAL: The 'request' argument must be string. Received type undefined

### ISSUE THREAD

**Issue by fabricioanciaes** [nuxt-community/router-extras-module#27]
I'm using nuxt with element-ui directly from the nuxt-create
added the router-extras module to my nuxt config:

```javascript
modules: [
		"@nuxtjs/axios",
		"@nuxtjs/pwa",
		"@nuxtjs/router-extras"
		[
			"nuxt-i18n",
			{
				//i18n stuff
			}
		]
	],
```

When I run yarn dev, I get `FATAL  The 'request' argument must be string. Received type undefined` and `✖ Nuxt Fatal Error. TypeError [ERR_INVALID_ARG_TYPE]: The 'request' argument must be string. Received type undefined`

I've only done this so far, did I mess up something or it's a bug?

I'm running nuxt 2.7.1

**Comment by ricardogobbosouza** [comment]
@fabricioanciaes Missing a comma
```js
modules: [
  "@nuxtjs/axios",
  "@nuxtjs/pwa",
  "@nuxtjs/router-extras", <--
   [
    "nuxt-i18n",
    {
      //i18n stuff
    }
  ]
],
```

**Comment by ubershmekel** [comment]
This error brought me here from google. Deleting `package-lock.json`, deleting the `node_modules` folder, and then `npm install` fixed it for me. This was during a migration from nuxt 2.8 to 2.10.

**Comment by ubershmekel** [comment]
I found myself here again with the exact same error. This time with Nuxt `"version": "2.12.2"` and my previous solution wasn't enough. I had to move my project folder outside of Dropbox. For some reason, the Dropbox meddling breaks the npm install and caused `The 'request' argument must be string. Received type undefined` error when I `npm run dev`.

I'm on Windows 10 Pro, and I npm installed from a cmd window.
